plugins/dotnet-auth-core-identity/src/core/create-seed-development-data.ts

```diff
@@ -34,66 +23,63 @@ export function CreateSeedDevelopmentDataBody(
           .Where(x => x.Value != null)
           .Select(x => x.Value.ToString())
           .ToArray();
-  
-          ${authEntityDto(authEntity, entities)}
-      
-      
-      if (!context.${entityNamePluralize}.Any(${entityFirstLetter} => ${entityFirstLetter}.UserName == ${entityNameToCamelCase}.UserName))
-      {
-          var password = new PasswordHasher<${name}>();
-          var hashed = password.HashPassword(${entityNameToCamelCase}, "password");
-          ${entityNameToCamelCase}.PasswordHash = hashed;
-          var userStore = new UserStore<${name}>(context);
-          await userStore.CreateAsync(${entityNameToCamelCase});
+
+          var user = new IdentityUser { Email = "test@email.com", UserName = "admin" };
+        
+ 
+          var password = new PasswordHasher<IdentityUser>();
+          var hashed = password.HashPassword(user, "password");
```

Comment:
This might be the reason why you struggled to login.

ASP.NET Core identity has a built-in validation for the password complexity. Try to use a password like `Password1!`

Actually, we could make username and password available from the appsettings.json for an easier user discovery

Reply:
@overbit  We updated the password value for now